doc/classes/MarginContainer.xml

```diff
@@ -30,16 +30,16 @@
 	</tutorials>
 	<theme_items>
 		<theme_item name="margin_bottom" data_type="constant" type="int" default="0">
-			All direct children of [MarginContainer] will have a bottom margin of [code]margin_bottom[/code] pixels.
+			Offsets towards the inside direct children of the container by this amount of pixels from the bottom.
```

Comment:
I agree with rephrasing, but I think the new one is still hard to parse.
It took me multiple attempts to break down in individual semantic elements and understand the flow of that sentence. "Offsets towards the inside" used as a verb especially makes it complex.

Reply:
Yes, that one is hard to get right. I could use "insets" perhaps, instead of the whole "offsets towards the inside", though I consider the "inset" word to be a bit less clear to non-English speakers for no particular reason.